app.py

```diff
@@ -9,4 +9,5 @@ def hello_world():
 
 
 if __name__ == '__main__':
-    app.run(debug=True)
+    app.debug=True
+    app.run(host='0.0.0.0')
```

Comment:
This had no problem earlier as well since we host it on heroku using gunicorn which manages the port forwarding itself. But it's a fine change.